woocommerce/woocommerce-android
# Remove update reader row when simulated reader disabled

### Please make sure #7948 is merged first. 

Closes: #7896 


### Description


To complement #7948, this PR removes the Update Simulated Reader Row from the Developer Options screen when the simulated reader option is disabled. 

### Testing instructions


1. Go to Menu - Settings - Developer Options
2. Enable and disable the Simulated reader toggle 
3. make sure the Update Simulated Reader Row is removed when the reader is disabled and added when the reader is enabled.

### Images/gif

<img src="https://user-images.githubusercontent.com/30724184/204318040-7faf63bc-2498-4cd8-835c-c4e8d5c8974c.gif" width="350"/>
